bazel/external/quiche.BUILD

```diff
@@ -2141,8 +2144,8 @@ envoy_cc_library(
 
 envoy_cc_library(
     name = "quic_core_http_spdy_stream_body_buffer_lib",
```

Comment:
Can you also rename this library to reflect the file name?

Reply:
Good catch, thank you.